src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsAutoListMapping.java

```diff
@@ -39,6 +35,18 @@ public static boolean shouldBeAutoListed(@Valid SscsCaseData caseData, Reference
             return isYes(overrideFields.getAutoList());
         }
 
+        if (Objects.equals(caseData.getBenefitCode(), "022")) {
```

Comment:
I would think you can do this in a single if statement:

```
if ("022".equals(caseData.getBenefitCode() ||"067".equals(...) || caseData.isIbcCase()) {
  return false;
}
```

But also, are those strings "022" / "067" available as enums anywhere?